src/main/java/org/broadinstitute/hellbender/tools/exome/CalculateTargetCoverage.java

```diff
@@ -660,7 +691,7 @@ protected CountColumns countColumns(final CalculateTargetCoverage tool) {
     protected enum Transform {
 
         /**
-         * Default read-count raw value (non-)transformation.
+         * Read-count raw value (non-)transformation.
```

Comment:
I think `Raw integer read count (non-)transformation` would make this a tad clearer.